test/spec/__snapshots__/pretty.test.js.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Pretty Logging Will pretty log an object in context 1`] = `
+"
+[37m[2018-05-18T16:25:57.815Z][39m [34mℹ info - hello world[39m
+
```

Comment:
that newline is not needed, have the object directly under the log line.

Reply:
I added that line to prevent this - http://prntscr.com/jr3he1